converter/__init__.py

```diff
@@ -1,6 +1,12 @@
-from .converter import celsius_to_fahrenheit, fahrenheit_to_celsius
+from builtins import ValueError
```

Comment:
@MariaMaximova, I think we don't need to import builtin ValueError here. Sorry for inconvenience for the previous comment about it :\

Reply:
done